internal/newsfeed/news_feed_manager.go

```diff
@@ -49,6 +50,12 @@ func WithHandler(handler FeedHandler) Option {
 	}
 }
 
+func WithLogger(logger *zap.Logger) Option {
+	return func(nfm *NewsFeedManager) {
+		nfm.logger = logger
```

Comment:
```suggestion
		nfm.logger = logger.Named("fetchRSS")
```
Then you wouldn't need to remember about the prefix in the logs anymore.